students/ABartles/Exercise2/Series.py

```diff
@@ -0,0 +1,41 @@
+"""
+Return the fibonacci or lucas series of n length.
+
+functions:
+fibonacci(n) - Return the fibonacci series of n length.
+lucas(n) - Return the lucas series of n length.
+sum_series(n, x, y) - Generlization used by fibonaci(n) and lucas(n).
+"""
+
+def fibonacci(n):
+    """Return the fibonacci series of n length starting with [0,1]."""
+    sum_series(n, 0, 1)
```

Comment:
Works well - assignment was to return the nth value, so looking for a single number.